cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -109,13 +112,22 @@ def test_snapshot_crud(small_test_vm, provider):
         casecomponent: Infra
         initialEstimate: 1/6h
     """
+    result = LogValidator(
```

Comment:
_Optional_ consider using the `LogValidator`'s contextmanager

Reply:
@john-dupuy trying if it could work, but I am losing wait="60s" in result.validate( ) by using contextmanager

Reply:
That's cool with me, left it _optional_ just to see if you wanted to implement it. If it doesn't work in this scenario, there's no need to shoehorn it in :+1: